target/linux/ramips/dts/mt7621_netgear_r6800.dts

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "mt7621_netgear_sercomm_bzv.dtsi"
+
+/ {
+	compatible = "netgear,r6800", "mediatek,mt7621-soc";
+	model = "Netgear R6800";
+
+	aliases {
+		led-boot = &led_power_white;
+		led-failsafe = &led_power_orange;
+		led-running = &led_power_white;
+		led-upgrade = &led_power_orange;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_orange: power_orange {
+			gpios = <&gpio_expander 0 GPIO_ACTIVE_LOW>;
+			label = "r6800:orange:power";
+		};
+
+		led_power_white: power_white {
+			gpios = <&gpio_expander 1 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:power";
+		};
+
+		led_wan_orange: wan_orange {
```

Comment:
Okay, so now you can remove all node labels you do not use, i.e. all except power...